htdocs/product/stock/tpl/stockcorrection.tpl.php

```diff
@@ -78,8 +78,8 @@ function init_price()
 	}
 	print img_picto('', 'stock').$formproduct->selectWarehouses($ident, 'id_entrepot', 'warehouseopen,warehouseinternal', 1, 0, 0, '', 0, 0, null, 'minwidth100');
 	print ' &nbsp; <select class="button buttongen" name="mouvement" id="mouvement">';
-	print '<option value="0">'.$langs->trans("Add").'</option>';
-	print '<option value="1"'.(GETPOST('mouvement') ? ' selected="selected"' : '').'>'.$langs->trans("Delete").'</option>';
+	print '<option value="3">'.$langs->trans("Add").'</option>';
```

Comment:
Why renaming 0 into 3.
This seems to break compatibility/feature with the page that include the tpl and manage the "correct_stock" action that need 0 or 1 to know if it is a + or - stock movement.